backend/app/resources.py

```diff
@@ -52,21 +52,22 @@ def checkout(user: User, resource_title):
             return jsonify({"msg": "Project not found"}), 404
         
         # Check to see if user has access to project
-        if username not in project["users"]:
+        users = project["users"]
+        if username not in users:
             return jsonify({'msg': 'User does not have access to project'}), 401
         # If we have enough available to check out 
-        if resource["availability"] < amount:
+        if int(resource["availability"]) < int(amount):
```

Comment:
What is the meaning of storing these numbers as strings?
I am open if the API works the same, but in this case I get `availability` as strings in GET `/resources` after checkin/checkout, causing more problems.

```
[
	{
		"_id": {
			"$oid": "654ebdcd626a7e314b0af231"
		},
		"availability": "0",
		"capacity": 100,
		"created_at": {
			"$date": "2023-11-10T23:33:33.040Z"
		},
		"title": "HW Set 1"
	},
	{
		"_id": {
			"$oid": "654ebdcd626a7e314b0af232"
		},
		"availability": 200,
		"capacity": 200,
		"created_at": {
			"$date": "2023-11-10T23:33:33.040Z"
		},
		"title": "HW Set 2"
	},
	{
		"_id": {
			"$oid": "654ebdcd626a7e314b0af233"
		},
		"availability": 300,
		"capacity": 300,
		"created_at": {
			"$date": "2023-11-10T23:33:33.040Z"
		},
		"title": "HW Set 3"
	}
]
```